src/js/stores/MesosStateStore.js

```diff
@@ -255,17 +255,21 @@ class MesosStateStore extends GetSetBaseStore {
     // the scheduler tasks or a list of Marathon application tasks.
     return frameworks.reduce(function (serviceTasks, framework) {
       let {tasks = [], completed_tasks = {}, name} = framework;
+
+      let allTasks = tasks.concat(completed_tasks).map(function (task) {
+        task.startedBy = name;
```

Comment:
I'm not to sure about the property name. `scheduledBy` or `scheduler` might be a better name. #nitpick